Makefile

```diff
@@ -52,9 +53,12 @@ release: setup-release
 install:
 	@luarocks make OPENSSL_DIR=$(OPENSSL_DIR) CRYPTO_DIR=$(OPENSSL_DIR)
 
-dev:
+remove:
 	-@luarocks remove kong
-	@luarocks make OPENSSL_DIR=$(OPENSSL_DIR) CRYPTO_DIR=$(OPENSSL_DIR)
+
+reinstall: remove install
```

Comment:
The PR and commit messages seem to indicate that they add a single Make target, but it seems they more than one: `make dev-only`, `make remove` & `make reinstall`.